packages/main/src/AvatarGroup.ts

```diff
@@ -374,8 +374,13 @@ class AvatarGroup extends UI5Element {
 	get _overflowButtonEffectiveWidth() {
 		const button = this._customOverflowButton ? this._customOverflowButton : this._overflowButton;
 		// if in "Group" mode overflow button size is equal to the offset from second item
+
+		if (!button) {
+			return 0;
+		}
+
 		if (this._isGroup) {
-			let item = this.items[1];
+			let item: HTMLElement = this.items[1];
```

Comment:
In this case setting the item of type Avatar will not work, because on line 388 we check:

```js
if (!item || item.hidden) {
     item = button;
}
```
so the item will no longer be of that type.

Reply:
I see, ok.